apps/testing-todos-list/src/app/list/ticket.store.spec.ts

```diff
@@ -16,7 +52,10 @@ describe('TicketStore', () => {
 
     describe('Given users api returns failure response', () => {
       it('Then tickets should not have any assignee', () => {
-        //
+        // I think this is the test that will help with line 50
+        // Not really sure how I would test this as Thomas' solution has a more involved setup function
+        // mockBackendService.users.and.returnValue(of(throwError()=> new Error("Blah blah blah"))) or something close
+        // then check tickets are the same as TICKETS
```

Comment:
did you push everything? There is no test here!!!

Reply:
Pseudo code.  Didn't write a test.